src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -339,6 +339,40 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
         createCallback("${COMMAND_PACKAGE}.PullImageResultCallback")
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    def createLoggingCallback() {
+        Class callbackClass = loadClass("${COMMAND_PACKAGE}.LogContainerResultCallback")
+        def delegate = callbackClass.getConstructor().newInstance()
+
+        Class enhancerClass = loadClass('net.sf.cglib.proxy.Enhancer')
+        def enhancer = enhancerClass.getConstructor().newInstance()
+        enhancer.setSuperclass(callbackClass)
+        enhancer.setCallback([
+
+            invoke: {Object proxy, java.lang.reflect.Method method, Object[] args ->
+                if ("onNext" == method.name && args.length && args[0]) {
+                  def frame = args[0]
+                  switch (frame.streamType as String) {
+                    case "STDOUT":
+                    case "RAW":
+                        System.out.print(new String(frame.payload))
```

Comment:
logger.quiet will be shown by default. logger.info will only be shown if the `-i` flag is given. This is why we have a simple logger.quiet message for each task to give some indication something has happened. Is this not happening in your case?